bessctl/module_tests/random_split.py

```diff
@@ -30,17 +30,17 @@
 from test_utils import *
 
 
-class BessRandomDropTest(BessModuleTestCase):
+class BessRandomSplitTest(BessModuleTestCase):
 
     def test_dropall(self):
```

Comment:
While this is not a part of this PR, but noticed that the names "dropall" and "dropnone" are actually swapped...?

Reply:
Fixed.